specification/columns/contractappliedcost.md

```diff
@@ -0,0 +1,41 @@
+# Contract Applied Cost
+
+Contract Applied Cost represents the cost of the charge applied to the contract.  Contract Applied Cost is applied to the contract via [Contract ID](#contractid).  Contract Applied Cost is commonly used for monitoring the progress towards fulfilling contractual commitments that facilitate discounts for [*resources*](#glossary:resource) or [*services*](#glossary:service) as negotiated between a provider and a customer.
+
+The ContractAppliedCost column adheres to the following requirements:
+
+* ContractAppliedCost MUST be present in a [*FOCUS dataset*](#glossary:FOCUS-dataset) when the provider supports *negotiated discounts*.
```

Comment:
Shouldn't it only be required when there's a commitment-tracking mechanism? Someone may offer negotiated discounts where everything applies and there's no need to track what was or wasn't applied, right?